Group3

nduvieilh opened the pull request:
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

1. elasticached-memcached
 elasticbeanstalk-environments  - no data found so removed from PR; combined with elasticbeanstalk-instances
 elasticbeanstalk-instances  - no data found so removed from PR; combined with elasticbeanstalk-environments
2. elasticsearch
3. elb
4. emr
fsx - no data found so removed from PR
glue - no data found so removed from PR
5. iam

Health infra template not included as it's not migrated to dimensional metrics.

<!-- DON'T DELETE THIS SECTION BELOW IF SUBMITTING A NEW QUICKSTART -->
## Pre merge checklist

<!-- This CHECKLIST SHOULD BE SUBMITTED FULLY COMPLETE WITH THE PR. IF NOT COMPLETE
THE PR REVIEW WILL BE DELAYED -->

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

Commit:
Added group of dashboards for aws

nduvieilh commented:
# Dashboard 1 - elasticached-memcached

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [x] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [x] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 2 - elasticbeanstalk-env

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [ ] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [x] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 3 - elasticbeanstalk-instances

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [ ] Find a infra dashboard with data, copy its providerAccountId
  - [x] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [ ] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 4 - elasticsearch

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [x] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [x] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [ ] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 5 - elb

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [x] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [x] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 6 - emr

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [x] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [x] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 7 - fsx

## 1. Validate dashboard
  - [ ] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [ ] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [x] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 8 - glue

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [ ] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [ ] Remove the `*_validate.json` file from the dashboard directory
  - [ ] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [ ] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 9 - health - DASHBOARD TEMPLATE MAY NOT BE MIGRATED TO DIMENSIONAL METRICS

## 1. Validate dashboard
  - [ ] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [ ] Find a infra dashboard with data, copy its providerAccountId
  - [ ] Add a providerAccountId to the filter on the migrated dashboard
  - [ ] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [ ] All widget titles are sentence case
  - [ ] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [ ] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [ ] `inventory` widget(s) are removed (if applicable)
  - [ ] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [ ] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [ ] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [ ] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [ ] Documentation link
  - [ ] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [ ] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [ ] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [ ] Reference to the dashboard(s)
## 7. Clean up
  - [ ] Remove the `*_validate.json` file from the dashboard directory
  - [ ] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [ ] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

# Dashboard 10 - iam

## 1. Validate dashboard
  - [x] Copy the `*_validate.json` to an account
    - If the accountId does not have data, find an account ID that does, replace the accountId properties with the new acountId
## 2. Compare against infra-dashboard
  - [x] Find a infra dashboard with data, copy its providerAccountId
  - [x] Add a providerAccountId to the filter on the migrated dashboard
  - [x] Document differences
    - If changes are basic, make them directly to the validate template.
    - If making changes, transfer these changes from the `*_validate.json` to the `*.json`.
## 3. Validate expected differences
  - [x] All widget titles are sentence case
  - [x] "Dashboard version x.y.z" is removed from markdown (if applicable)
  - [x] Remove `::: beyond` and `:::` from the beginning and end of markdown `text` property.
  - [x] `inventory` widget(s) are removed (if applicable)
  - [x] `event-feed` widget(s) are removed (if applicable)
## 4. Take a screenshot
  - [x] Take a screenshot of the dashboard
    - Make sure it does not include any PII
  - [x] Add screenshot to dashboard directory as a `*.png` file
    - Rename image files to match JSON file name. Add a number if including multiple screenshots, i.e. `*1.png` and `*2.png`.
## 5. Add config.yml
  - [x] Validate if a `quickstarts/**/*` directory/file exist
    - If not, move the one from the dashboard directory
    - If so, compare it to the one in the dashboard directory; make changes as necessary
## 6. Confirm contents of config.yml
  - [x] Documentation link
  - [x] Appropriate `installPlan`
    - Check documentation link for correct plan type.
  - [x] Logo 
    - Preferably a `logo.svg` file in the same directory. If using icons from [AWS Assets package](https://aws.amazon.com/architecture/icons/), use light theme / size 64 if available.
  - [x] Description
    - Can be generic. Correct service title to be human-readable by uppercasing any acronyms.
  - [x] Reference to the dashboard(s)
## 7. Clean up
  - [x] Remove the `*_validate.json` file from the dashboard directory
  - [x] Remove the `config.yml` file from the dashboard directory
## 8. Move the PR from draft
  - [ ] Create the actual PR, ensure all PR **Pre merge checklist** items are complete
  - [ ] Update Confluence list of migrated templates [Confluence](https://newrelic.atlassian.net/l/cp/mHMXBqjx)

lroyce commented:
## Dashboard 1 - elasticached-memcached

###Dashboards:
There was no previous existing elasticached-memcached json template, so subdir `newrelic-quickstarts/dashboards/aws-elasticached-memcached/` was created.

Infra-dashboard-template had an incorrect query for "Current connections"

###Quickstarts
There was a previously existing elasticache config that should point to both memcache and redis elasticache jsons.

Account used to verify: OHAI

lroyce committed:
feat(group3-validate-aws-elasticachememcached): Validated template for elasticache-memecached.

Updated Cache hit % query as it had hanging WHERE

Updated config yaml in quickstarts aws-elasticach

lroyce commented:
## Dashboard 2 - elasticbeanstalk-env - REMOVED DUE TO NO DATA IN DASHBOARD SCREENSHOTS

Also, beanstalk env and beanstalk instances were combined into a tabbed dashboard experience.

### Dashboards: 
There was no previously existing newrelic-quickstart dashboard template, so one was created at `newrelic-quickstart/dashboards/aws-elasticbeanstalk-environments/ 

I could not find an account with elasticbeanstalk-environment test data so the screenshots are somewhat empty of data. 

### Quickstarts:
There was a previously existing newrelic-quickstart quickstart. So I updated the dashboard and added a keyword.

lroyce committed:
feat(group3-validate-elasticbeanstalk-environments): Validated the migrated json template.

Could not find an account that had data so screenshots are empty.
Updated quickstart. Logo matched so not updated.

lroyce committed:
feat(group3-validate-elasticbeanstalk-instances): Validated migrated json template.

Could not find account with data to compare against.
Logo was the same. Updated config to point to this dashboard as well.

lroyce commented:
## Dashboard 3 - elasticbeanstalk-instances - REMOVED DUE TO NO DATA IN DASHBOARD SCREENSHOTS

Also, beanstalk env and beanstalk instances were combined into a tabbed dashboard experience.

### Dashboards:
There was no previously existing newrelic-quickstart dashboard template, so one was created at `newrelic-quickstart/dashboards/aws-elasticbeanstalk-instances/

I could not find an account with elasticbeanstalk-instances test data so the screenshots are somewhat empty of data.

### Quickstarts:
There was a previously existing newrelic-quickstart quickstart. So I updated the dashboard and added a keyword. Logo was correct so didnt update that.

lroyce commented:
## Dashboard 4 - elasticsearch-service

### Dashboards:
There was a previously existing dashboard template at `dashboards/elasticsearch` so an additional subdirectory was made at `newrelic-quickstarts/dashboards/aws-elasticsearch` for `aws-elasticsearch.json`.

The difference between the two are mostly with the queries. IO includes the widget "Search Rate" that infra does not. *"Average CPU utilization (%)" and "Max jvm memory pressure (%)": Infra's query is "byCluster" and IO's query is "byNode".
*"Automatic snapshot failure" and "Max cluster used space (GiB)": IO's query for both included ".Maximum"
*"Min free storage space (GiB)": IO's has ".Minimum" on it.

**UPDATE** I realized by creating a IO dashboard using the previously existing template that none of the queries work due to the inclusion of the cloudwatch metric stream Where clause. So I deleted the `elasticsearch.json` and replaced it with the aws-elasticsearch.json.

Current IO Template
![Screen Shot 2022-09-21 at 5 21 06 PM](https://user-images.githubusercontent.com/33464917/191632312-42ad10d0-4775-4f9a-888e-e4c27f028bc5.png)

### Quickstarts:
There is a previously existing quickstart pointing to the `elasticsearch.json`. I added `aws-elasticsearch.json`. 

Account used for verification: Beyond

lroyce committed:
feat(group3-validate-aws-elasticsearch): Validated elasticsearch template.

There is a previously existing IO json template. Queries on a few widgets are different.
Added aws-elasticsearch to the previously existing quickstart config. Logo is the same so did not update.

lroyce committed:
feat(group3-validate): Updated migrated templates that have markdown formatting issues by removing '::: beyond'

lroyce commented:
## Dashboard 5 - elb

### Dashboards:
There was no previously existing dashboard template, so `newrelic-quickstart/elb/aws-elb.json` was migrated in. 

Queries were affect the regex search so added back in `AND` and removed a `WHERE`. 

### Quickstarts:

Updated the `installPlan` per the documentation. Logo was good.

lroyce committed:
feat(group3-validate-elb): Updated template json, some queries needed updating. Updated quickstart.

lroyce commented:
## Dashboard 6 - emr

### Dashboards:
There was no previously existing dashboard template, so `newrelic-quickstart/elb/aws-elb.json` was migrated in. 

Queries were correct except for the query for 'Cluster jobs status (hadoop 1)', it returned no data no matter what. 

### Quickstarts:

Updated the `installPlan` per the documentation. Logo was good.

lroyce committed:
feat(group3-validate-emr): Validated json template, one query not retreiving data but nothing incorrect.

Quickstart previously existed, so updated with correct dashboard.

lroyce committed:
feat(group3-validate-all): Updating previously validated jsons by uppercasing acronyms

lroyce commented:
## Dashboard 7 - fsx - REMOVED DUE TO NO DATA IN DASHBOARD SCREENSHOTS

### Dashboards:
There was no previously existing dashboard template, so `newrelic-quickstart/elb/aws-fsx.json` was migrated in. Upper cased acronyms and camel cased widget titles as they relied heavily on naming metrics.

### Quickstarts:

There was a previously existing quickstart, I added a `dashboards` property and pointed it to `aws-fsx.json`, confirmed logo was correct, and updated `installPlans` to aws-cloudwatch-metrics-streams

lroyce committed:
feat(group3-validate-fsx): Validated template json, uppercased and camelcased names where appropriate.

Config existed already, updated dashboards, and installPlans.

lroyce commented:
## Dashboard 8 - glue - REMOVED DUE TO NO DATA IN DASHBOARD SCREENSHOTS

### Dashboard
There was no previously existing IO template json so `newrelic-quickstart/elb/aws-glue.json` was migrated in. Updated the acronyms to uppercase.

Changed coordinates of the last two widgets to fit better into rows.

### Quickstarts
There was a previously existing quickstart. I updated the installPlans to cloudwatch metric streams, added the dashboard to point to `aws-glue.json` and confirmed the logo was correct.

lroyce committed:
feat(group3-validate-glue): Validated dashboard json template, could not find an account with data. Updated acronyms to uppercase. Quickstart existed, added dashboard and updated installPlan.

lroyce committed:
feat(group3-validate-health): Not validated due to not being migrated to dimensional metrics, removing migrated subdir aws-health

lroyce committed:
feat(group3-validate-iam): Validated json, changed sizing on markdown widget.

Quickstart config existed already, updated installPlan, added dashboard and confirmed logo.

lroyce commented:
## Dashboard 9 - health

The dashboard's metrics have not been migrated to dimensional metrics. Skipping this and removing `aws-health` subdir from the PR.

lroyce commented:
## Dashboard 10 - iam

### Dashboard
No previously existing dashboard so `newrelic-quickstart/elb/aws-iam.json` migrated in. 

Changed the height of markdown widget. Upper cased json title too. 

### Quickstart
There was a previously existing quickstart. Confirmed configs contents, updated `installPlans`, added `dashboards` property and confirmed logo is correct.

lroyce committed:
feat(group3-validate): Removing config that I forgot to delete.

lroyce committed:
feat(group3-validate-aws-elasticsearch): Removing old IO amazon elasticsearch.

Template no longer works on all accounts. Replacing with aws-elasticsearch

lroyce committed:
Updated elasticache memcached screenshot

lroyce committed:
Merge branch 'main' into group3

lroyce committed:
fix(group3-validate-elasticache-memcached): Updated template

Renamed directories to amazon-elasticache-memcached, updated config to reflect.

lroyce committed:
fix(group3-validate-amazon-elasticsearch): Updated template
Renamed from aws to amazon, update config and screenshot names as well.

lroyce committed:
fix(group3-validate-aws-emr): Update template
Updated name from aws to amazon, including config and image names.

lroyce committed:
fix(group3-aws-elasticbeanstalk): Removed elastic beanstalk
from this PR due to not having data from screenshots, removed screenshots, json, updated config to
not have dashboard.
Also, these templates will be combined into a single dashboard once data is found.

lroyce committed:
fix(group3-validate-fsx): Update template name
from aws to amazon. Removed this template due to no data in screenshots.

lroyce committed:
fix(group3-validate-glue): Removed template
from PR due to no data in screenshots. Updated config yml

lroyce committed:
fix(group3-validate-iam): Updated markdown
widget to have correct aws title.

lroyce committed:
fix(group3-validate-elasticache): Updated name
of quickstart to amazon-elasticache

lroyce committed:
Merge branch 'newrelic:main' into group3

lroyce committed:
feat(group3-markdown-update): Removed time reporting indications in
markdown.

lroyce committed:
Merge branch 'group3' of https://github.com/nduvieilh/newrelic-quickstarts into group3
